internal/rsyncd/match.go

```diff
@@ -243,12 +243,14 @@ func (st *sendTransfer) matched(h hash.Hash, f *os.File, head rsync.SumHead, off
 			offset, st.lastMatch, i, head.Sums[i].Len, n)
 	}
 
+	/* FIXME: this is not used
 	l := int64(0)
 	if !transmitAccumulated {
 		l = head.Sums[i].Len
 	}
+	*/
```

Comment:
I left this code here, do you think it should be removed?

Reply:
I’d say let’s leave it here for the next time someone works on this part of the algorithm. They can always remove it if they don’t find it helpful :)